ci/reload-env.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-
+set -e
+set -x
```

Comment:
Github actions put `set -e` as a default option. `set -x` will add unnecessary call records and should be used for debug purpose only.

Reply:
`set -x` can help us locate bugs, I prefer enable it. For CI, I think it's OK to have more logs to keep as much info as possible. Is there any privacy issue for enable it?

Reply:
When using `if` block in bash, option `-e` will output EVERY statement executed. Thus it is not appropriate to enable this globally. I'll remove it myself.